src/test/thrift/test.thrift

```diff
@@ -19,7 +19,7 @@ enum Enum {
 }
 
 struct StructB {
-  1: required Union u
+  1: required Union1 u
```

Comment:
I had to rename this because it seemed to be having a name clash with an internal class called `Union` also!